packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1061,6 +1067,10 @@ class EditableText extends StatefulWidget {
   /// {@template flutter.widgets.editableText.onSubmitted}
   /// Called when the user indicates that they are done editing the text in the
   /// field.
+  ///
+  /// When the user has finalized editing, by default [onSubmitted] is called after
+  /// [onChanged] or after [onEditingComplete] when the default behaviour is overridden.
+  /// See [onEditingComplete] for details.
```

Comment:
Maybe: 
```
/// By default, [onSubmitted] is called after [onChanged] when the user 
/// has finalized editing; or, if the default behavior has been overridden,
/// it is called after [onEditingComplete]. See [onEditingComplete] for
/// details.
```

Also minor nit, behaviour -> behavior. I'm guilty of adding hundreds of non-US spellings to our codebase myself, and occasionally [do passes](https://github.com/flutter/flutter/pull/33323) over the code to fix all the ones I missed :wink:

Reply:
Ohh thanks so much, I will keep it in mind for future doc PRs)